confd/service/service.go

```diff
@@ -0,0 +1,188 @@
+package service
+
+import (
+	"encoding/json"
+	"html/template"
+	"log"
+	"os"
+	"path"
+
+	. "github.com/cloudogu/ces-confd/confd"
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
should not use dot imports